src/Favicon/Favicon.php

```diff
@@ -185,7 +185,10 @@ private function getFavicon($url, $checkDefault = true) {
 
         // See if it's specified in a link tag in domain url.
         if (!$favicon) {
-            $favicon = $this->getInPage($url);
+            $favicon = trim($this->getInPage($url));
+        }
+        if (substr($favicon, 0, 2) === '//') {
+            $favicon = 'https:' . $favicon;
```

Comment:
Good idea. However, it might be a better idea to either use `http://` or the protocol where the favicon was found, because HTTPS might not be available.